### ISSUE TITLE
item/block display entities

### ISSUE THREAD

**Issue by ghost** [GeyserMC/Geyser#3810]
### Describe the bug

Display entities are not visible and give a warning on console

### To Reproduce

Bug was first discovered using https://www.spigotmc.org/resources/cultivation.109919/
1. Place an item
2. Item disappears and error is seen

### Expected behaviour

Display entities should be visible - Bedrock side and warning should not be given

### Screenshots / Videos

![image](https://github.com/GeyserMC/Geyser/assets/97077949/aa7efd3e-ad80-4adf-92fd-fb3092993e75)

### Server Version and Plugins

  [Sat 07:24:04 INFO ] Checking version, please wait...
  [Sat 07:24:04 INFO ] Current: git-Purpur-1967 (MC: 1.19.4)*
Previous: git-Purpur-1957 (MC: 1.19.4)
* You are 13 version(s) behind
Download the new version at: https://purpurmc.org/downloads

### Geyser Dump

https://dump.geysermc.org/vmOqUykQvaDGl0ZqOf0RrHssDXhX2WaI

### Geyser Version

  [Sat 07:24:31 INFO ] Geyser-Spigot version 2.1.0-SNAPSHOT   [Sat 07:24:31 INFO ] Website: https://geysermc.org   [Sat 07:24:31 INFO ] Author: GeyserMC

### Minecraft: Bedrock Edition Device/Version

1.19.83
windows 10 operator

### Additional Context

N/A

**Comment by xPOLT3RG3ISTx** [comment]
So what is the fix? Yalll keep adding threads to this but not telling us the fix.

**Comment by Camotoy** [comment]
The fix is very complicated, because block and item display entities do not exist in Bedrock. So we have to recreate them from scratch basically.
Also, quaternions suck and I do not want to start messing with converting quaternions to vectors which is what we will have to do.

**Comment by Camotoy** [comment]
I'm not aware of anyone who is working on a fix, or has a desire to work on a fix.

**Comment by Kas-tle** [comment]
A lot of things would need to come first for us to properly address this. We'd need to first get entity properties working which we don't really deal with at the moment. We'd also need to add to geyser optional pack some way of taking rotation/translation/scale values and interpolating between those two states over a given time period as java does. And we'd need to implement a whole new entity and figure out the base translations for items/blocks in hand. So yeah while our goal is to do all this eventually it will take time and like all geyser features there is no ETA.

**Comment by xPOLT3RG3ISTx** [comment]
Ah okay. Well is this affecting our servers negatively or just purely a annoying server console popup?

**Comment by onebeastchris** [comment]
Well.. Bedrock players just won't see Item and Block display entities, that's all

**Comment by anar4732** [comment]
any update?

**Comment by onebeastchris** [comment]
Yes, only one; you won't get the spam about unknown entities. That's been moved to debug level.

As for actual support on display/item entities, we do not have an eta - Bedrock edition does not have them natively, so we would have to hack around that. That takes time to implement. 

We'll close this issue when that's done, for now, you won't see those entities on Bedrock edition.
